internal/buildscripts/packaging/msi/make.ps1

```diff
@@ -36,6 +36,7 @@ function Install-Tools {
 
     choco install wixtoolset -y
     setx /m PATH "%PATH%;C:\Program Files (x86)\WiX Toolset v3.11\bin"
+    echo "C:\Program Files (x86)\WiX Toolset v3.11\bin" | Out-File -FilePath $env:GITHUB_PATH -Encoding utf8 -Append
```

Comment:
This line is required to add the WiX Toolset to the path on GitHub Actions. Once CircleCI is removed, the above line: "setx /m ..." will need to be removed.